### PR TITLE
fix(A11y): ToggleSwitchField color contrast issues [KZN-1156]

### PR DESCRIPTION
**Pull request by christian314159**
## Why
- The color contrast on our ToggleSwitch component is insufficient
Fixes cultureamp/kaizen-discourse#72

## What
- darkened colors to fix contrast
- adding check icon to ON state to make it clearer w/o colors

### CODE REVIEW

File: draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss [hunk 1]
```diff
@@ -57,10 +57,20 @@ $focus-ring-offset: 1px;
...
 }
 
+.checkIcon {
+  color: $color-green-500;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin: -10px 0 0 -9px;
+  opacity: 0;
+  transition-duration: 0.2s;
```
Comment (dougmacknz) [inline draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss:67]:
```suggestion
  transition-duration: $animation-duration-rapid;
```

File: draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss [hunk 2]
```diff
@@ -57,10 +57,20 @@ $focus-ring-offset: 1px;
   transition: left $animation-timing, right $animation-timing;
 }
 
+.checkIcon {
+  color: $color-green-500;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin: -10px 0 0 -9px;
```
Comment (dougmacknz) [inline draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss:65]: Can we use `text-align: center` or flex to center the icon?

File: draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss [hunk 3]
```diff
@@ -80,10 +90,16 @@ $focus-ring-offset: 1px;
     }
   }
 
+  .checkIcon {
+    opacity: 1;
+    transition-delay: 0.1s;
```
Comment (dougmacknz) [inline draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss:95]:
```suggestion
    transition-delay: $animation-timing;
```

File: draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss [hunk 4]
```diff
@@ -80,10 +90,16 @@ $focus-ring-offset: 1px;
...
+    transition-duration: 0.3s;
```
Comment (dougmacknz) [inline draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss:96]:
```suggestion
    transition-duration: $animation-duration-fast;
```

File: package.json [hunk 5]
```diff
@@ -37,13 +37,13 @@
...
     "@kaizen/design-tokens": "^10.0.11",
     "@kaizen/tailwind": "*",
-    "@storybook/addons": "^7.0.5",
-    "@storybook/api": "^7.0.5",
-    "@storybook/blocks": "^7.0.5",
-    "@storybook/components": "^7.0.5",
-    "@storybook/core-events": "^7.0.5",
-    "@storybook/react": "^7.0.5",
-    "@storybook/theming": "^7.0.5",
+    "@storybook/addons": "^7.0.7",
```
[the remark this answers is not in the document]
Comment (christian314159) [reply package.json:40]: Oops, but I think  that'll be fine once I rebase.

File: draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss (hunk 2 above)
Comment (christian314159) [reply draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss:65]: There might be a different way, but `text-align` doesn't work. With the icon component within the "dot" and also the check mark icons "visual center" not being its actual center, the positioning is a bit finicky.

File: .changeset/selfish-rules-cheer.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/draft-form": minor
+---
+
+Accessibility fix: improved color contrast for ToggleSwitchField
```
Comment (dougmacknz) [inline .changeset/selfish-rules-cheer.md:5]: Should probs mention adding the check

Comment (christian314159) [reply .changeset/selfish-rules-cheer.md]: Added

**Review by dougmacknz — APPROVED** [review]

**Pull request merged**